charts/schema-registry/values.yaml

```diff
@@ -29,6 +29,26 @@ service:
   type: LoadBalancer
   port: 9092
 
+initContainer: {}
+  # name: ""
+  # image:
+  #   repository:
+  #   tag:
+  #   pullPolicy: IfNotPresent
+  # command: []
```

Comment:
I'd add an example of the command to copy from the initcontainer to the location of the mountPath.

Reply:
done